Makefile-Tasking-Tricore/custom_compiler_yaml_tasking.py

```diff
@@ -0,0 +1,70 @@
+output_filename = "custom-compiler-tasking-generated.yaml"
+tasking_install_folder = "C:\\Program Files\\TASKING\\TriCore v6.3r1"
+tasking_platform_suffix = ["51", "arm", "mcs", "pcp", "tc"]  # TriCore, ARM
+
+c_additional_compile_switches = "--iso=99"  # example, may be empty
+
+cpp_additional_compile_switches = "--c++=11"  # example, may be empty
+
+import os, subprocess, tempfile, re
```

Comment:
I'd expect imports on the top

Reply:
well, I don't know... There are a bunch of constants which supposed to be adjusted by the user, I wanted put them to the very top of the file

Reply:
Moved, also some comments added